packages/hyperion-flowlet/src/Flowlet.ts

```diff
@@ -46,10 +46,7 @@ export class Flowlet<T extends FlowletDataType = FlowletDataType> {
       let prefix: string = f ? (f._fullName ?? '...') : '';
       for (let j = rawFlowlets.length-1; j>=0; --j){
         let flowlet = rawFlowlets[j];
-        flowlet._fullName = prefix = prefix + "/" + flowlet.name;
-        if (includeId){
-          flowlet._fullName += ":" + flowlet.id;
-        }
+        flowlet._fullName = prefix = prefix + "/" + flowlet.name + ":" + flowlet.id;
```

Comment:
With these changes, we are now using `:` to both append the flowlet id and add flowlet data details, e.g. `xhr(method:${method}&url:${request.uri.pathname})` for network responses. This flowlet's full name would then look something like `/xhr(method:POST&url:/api/graphql/):170287`. Wondering if we should use a different delimiter for the flowlet id (maybe `_`?) to make it easier to parse, e.g. in TA

Reply:
Great observation, since I replaced the `,` with `&` to be more URI like format, one option is to use `=` instead of `:` for flowlet data details. Then your example will look like `/xhr(method=POST&url=/api/graphql/):170287`, what do you think?